front/app/component-library/components/Tooltip/index.tsx

```diff
@@ -25,26 +25,90 @@ const useActiveElement = () => {
   };
 
   useEffect(() => {
-    document.addEventListener('focusin', handleFocusIn);
+    document.addEventListener('click', handleOutsideClick);
     return () => {
-      document.removeEventListener('focusin', handleFocusIn);
+      document.removeEventListener('click', handleOutsideClick);
     };
-  }, []);
+  });
 
   useEffect(() => {
-    document.addEventListener('click', handleOutsideClick);
+    document.addEventListener('focusin', handleFocusIn);
     return () => {
-      document.removeEventListener('click', handleOutsideClick);
+      document.removeEventListener('focusin', handleFocusIn);
     };
-  });
+  }, []);
 
   return active;
 };
 
+const TippyComponent = ({
```

Comment:
@amanda-anderson I was thinking of fixing the sense-making tooltip here as well but I saw you already completed this task. Anyhow, I took the opportunity to clean up the tooltip component a bit, we have a lot of code duplication. 

I didn't really change anything but had a question about the new `useWrapper` prop. I see it's only being used in the icon tooltip component but the component seems to work fine without it. I tested very quickly so I might have missed something but I was wondering, why did we introduce it in the first place? Do you remember?

Reply:
Yes it had to do with [Tippy accessibility](https://atomiks.github.io/tippyjs/v5/accessibility/) :thinking: We had an issue because in the past we always wrapped Tippy content in a <div> but this wasn't accessible when the content was a Button or Input (in this case they should be the direct children of Tippy - there was some client's accessibility audit were this came up).

If we just removed the <div> though from the Tooltip entirely, it broke/caused a visible crash when Tippy was wrapping something that wasn't a Button or Input. So we ended up adding this useWrapper behaviour instead to maintain the current behaviour but also let us fix the accessibility issues in the front office.

Reply:
I wonder what the exact problem was since we have mechanisms in this component to address all focus-related concerts. Or maybe they just looked at the HTML and decided it's not compliant? 🤔  Asking because the useWrapper prop false only in the IconButton so in any case feels like a very limited use case. In any case, happy to leave it as is, was just curious about it..

Reply:
>Or maybe they just looked at the HTML and decided it's not compliant

Indeed I think it was something like this, about the Html not being valid or something :thinking: